File: URI-ONLINE-JUDGE/Extremely Basic/1008.cpp
#include <iostream>
using namespace std;
int main()
{
    int NUMBER, WHOUR;
    float AMOUNTPH, SALARY;

    scanf("%d", &NUMBER);
    scanf("%d", &WHOUR);
    scanf("%f", &AMOUNTPH);

    SALARY = WHOUR * AMOUNTPH;

    printf("NUMBER = %d\n", NUMBER);
    printf("SALARY = U$ %.2f\n", SALARY);

    return 0;
}

File: URI-ONLINE-JUDGE/Extremely Basic/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var NUMBER = parseInt(lines.shift());
var WHOUR = parseInt(lines.shift());
var AMOUNTPH = parseFloat(lines.shift());

var SALARY = WHOUR * AMOUNTPH;

console.log('NUMBER = ' + NUMBER);
console.log('SALARY = U$ ' + SALARY.toFixed(2));